components/eamxx/src/share/io/scream_scorpio_interface.hpp

```diff
@@ -69,6 +69,8 @@ extern "C" {
   void set_int_attribute_c2f (const char*&& filename, const char*&& attr_name, const int& value);
   int get_dimlen_c2f(const char*&& filename, const char*&& dimname);
   bool has_variable_c2f (const char*&& filename, const char*&& varname);
+  /* Query a netCDF file for the time variable */
+  double read_time_at_index_c2f(const char*&& filename, const int& time_index = -1);
```

Comment:
How about not defaulting time index to -1, and instead offering `read_curr_time_c2f` as well? It could simply call `get_dimlen_c2f` and then call `read_time_at_index_c2f` with the current time length...

Reply:
Great suggestion.  Should I get rid of the option to even pass a negative or 0 value to this function, now that the `read_curr_time` is in?

Reply:
Yes, I would say no default for `time_index` anymore, and negative values should be considered an error.

Reply:
@AaronDonahue Are you going to change `read_time_at_index_c2f`, enforcing non-negative time index? Or do you want to leave it like this?